src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -63,49 +63,69 @@ object SbtSassCompiler extends AutoPlugin {
             -- (Assets / compileSass / excludeFilter).value
         )
         val sassFilesFound  = sourceDir.globRecursive(sassFilesFilter).get.filterNot(_.isDirectory)
-        val logger          = streams.value.log
-        logger.info(s"Sass compiling via sbt-sass-compiler: ${sassFilesFound.length} files")
+
+        val streamsValue = streams.value
+        val logger       = streamsValue.log
 
         // Assets / webModules unpacks webjars to the webJarsDirectory target/web-modules/main
         val sassLoadPaths = List[java.io.File](
           (Assets / webJarsDirectory).value
         ).asJava
 
-        Using(SassCompilerFactory.bundled()) { sassCompiler =>
-          val startInstant                                                   = System.currentTimeMillis
-          val eitherCompiledCssFiles: Seq[Either[Throwable, (String, Path)]] = sassFilesFound.map { sassFile =>
-            sassCompiler.setLoadPaths(sassLoadPaths) // no need to set a path of the current file as well
-            Try(sassCompiler.compileFile(sassFile).getCss) match {
-              case Success(compiledCss)      =>
-                val cssFile = targetPath
-                  .resolve(sourcePath.relativize(sassFile.toPath))
-                  .resolveSibling(sassFile.base + ".css")
-                Right((compiledCss, cssFile))
-              case Failure(compilationError) => Left(compilationError)
+        // Compilation function that will be called if uncompiled Sass detected on run of `sbt assets`
+        def compileSassToCssFiles() = {
```

Comment:
I know it's a method within another method (`projectSettings`), but I think it will be worth to add return type, just for better visibility and to be consistent with rest of the file

Reply:
@PatrykRudnicki Agreed, will add that now.